workflow/tests/test_documentation.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative '../../HPXMLtoOpenStudio/resources/minitest_helper'
+require 'rdoc/rdoc'
+
+class WorkflowDocumentationTest < Minitest::Test
+  def setup
+    @target_coverage = 0.0
+  end
+
+  def test_rdoc
+    folder_excludes = []
+    check_all = true
+
+    rdoc = RDoc::RDoc.new
+
+    folders = Dir.glob('*').select { |f| File.directory? f }
+    folders.each do |folder|
+      next if folder_excludes.any? { |f| f.include?(folder) }
+
+      rb_files = Dir[File.join(folder, '**/*.rb')]
+      next if rb_files.size == 0
+
+      argv = ['--dry-run', folder]
+      rdoc.document(argv)
+      stats = rdoc.stats
+      puts "#{folder}: #{stats.percent_doc.round(2)}%"
+      puts
+    end
+
+    if check_all
+      # Parse files
+      argv = ['--dry-run']
+      rdoc.document(argv)
+
+      # Check stats
+      stats = rdoc.stats
+      assert_operator(stats.percent_doc, :>, @target_coverage)
```

Comment:
Help me understand what's going on here. I thought rubocop will complain if a method is not documented. Why do we need a separate test using rdoc?

Reply:
We don't _need_ this test. I started this whole update with rdoc (where target coverage was set to 100%). Now that we've pursued rubocop, I kept this test just for summary/information purposes (setting target coverage down to 0%).